### ISSUE TITLE
Edit Orders / Give for comes up blank if order has stop date in past

### ISSUE THREAD

**Issue by dancunningham** [projectbuendia/client#209]
The “Give for” field doesn’t work how I would expect when editing an order that had a previous stop date in the past:
1. It comes up blank - it should say the number of days that the order was originally prescribed for
2. The guidance text and the dates that are set are incorrect. I have an order that was started on 21 October. If I set it to 3 days, it says "stop after 17 Jan", which is 3 days from now, not 3 days from 21 October. It should say "stop after 23 October".

This aspect of it seems to behave more like a way to extend an order, rather than a way of editing what was ordered.

We can work around this for the pilot, and after that I think this needs a closer look.

**Comment by capnfabs** [comment]
Yeah, this was an intentional design decision on @zestyping's part; there's a lot of code designed to do exactly this. The idea was (I think) that it's easier to think in terms of extending an order, and you're very rarely rewriting history, so being able to set a duration that ends in the past doesn't make sense.

Agreed that the UX for it doesn't make a lot of sense though.

**Comment by capnfabs** [comment]
Unassigning until I actually get close to working on it :)
